### PR TITLE
Remove openssh-server from base image

### PR DESCRIPTION
**Pull request by kenXengineering**
Added apt-get upgrade to upgrade image when built
Add ENV REFRESHED, update this to cause an image rebuild

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,8 @@
 FROM phusion/baseimage:0.9.16
 MAINTAINER Modulus <help@modulus.io>
 
+ENV REFRESHED_ON=12-15-2015
```
Comment (jackboberg) [inline Dockerfile]: Isn't this accomplished by just doing:

`docker build --no-cache`

Seems like an anti-pattern to add an environment variable just to force rebuild. I know you can use this while in dev to preserve upper layers, but for the published source?

**Comment by kenXengineering** [comment]
@jackboberg got it, removed it.  You have a good point with the --no-cache.

**Comment by abrongersma** [comment]
:+1:

**Comment by jackboberg** [comment]
:+1:

**Pull request merged**